UnityProject/Assets/Scripts/Systems/CraftingV2/RelatedRecipe.cs

```diff
@@ -18,7 +18,19 @@ public class RelatedRecipe
 		/// <summary>
 		/// 	The ingredient's index in a recipe's required ingredients list.
 		/// </summary>
-		public int IngredientIndex => ingredientIndex;
+		public int IngredientIndex
+		{
+			get
+			{
+				return ingredientIndex;
+			}
+#if UNITY_EDITOR
```

Comment:
why is it important make this set only available in editor?

Reply:
Changing it not in unity editor doesn't make sense and should be forbidden because otherwise it's an easy way to create bugs

I hope if someone will have tried to change value during runtime will get an exception